database/frontend/animations.json

```
@@ -131,5 +131,12 @@
     "url": "https://www.react-spring.dev/",
     "category": "frontend",
     "subcategory": "animations"
+  },
+  {
+    "name": "GSAP",
+    "description": "GSAP is a robust industry standard JavaScript animation library from GreenSock that lets you craft high-performance animations that work in every major browser. Animate CSS, SVG, canvas, React, Vue, WebGL, colors, strings, motion paths, generic objects...anything JavaScript can touch!",
+    "url": "https://gsap.com/",
+    "catagory": "frontend",
+    "subcatagory": "animations"
   }
```

[proofreading] @psykat1116, I figured out why the deployment failed...formatting issues. Add a comma after the closing bracket.   
```suggestion
    "name": "GSAP",
    "description": "GSAP is a robust industry-standard JavaScript animation library from GreenSock that lets you craft high-performance animations that work in every major browser. Animate CSS, SVG, canvas, React, Vue, WebGL, colors, strings, motion paths, generic objects...anything JavaScript can touch!",
    "url": "https://gsap.com/",
    "category": "frontend",
    "subcategory": "animations"
  }, 
```